tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -1460,11 +1461,8 @@ def test_slow_unfinished_sequence(self):
         self.assertEqual(
             out,
             {
-                "chunks": [
-                    {"text": "", "timestamp": (18.94, 0.02)},
```

Comment:
This didn't make sense before as the audio is only 4 seconds long. Seems like changing from `forced_decoder_inputs` to `decoder_input_ids` correts this actually.